src/electionguard/encrypt.py

```diff
@@ -35,18 +35,39 @@ class EncryptionDevice(Serializable):
     """
 
     uuid: int
+    """Unique identifier for device"""
+
+    session_id: str
+    """Used to identify session and protect the timestamp"""
+
+    launch_code: int
+    """Election initialization value"""
+
     location: str
+    """Arbitary string to designate the location of device"""
 
-    def __init__(self, location: str) -> None:
+    def __init__(
+        self,
+        session_id: str,
+        launch_code: int,
+        location: str,
+    ) -> None:
         self.uuid = generate_device_uuid()
         self.location = location
+        self.launch_code = launch_code
```

Comment:
ctor order match the hash order

Reply:
I'm down with this idea but counter intuitive with the notes on uuid.